packages/shared/src/dto/workflows/step.dto.ts

```diff
@@ -42,14 +42,17 @@ interface Issue<T> {
 }
 
 export class StepIssuesDto {
-  body?: Record<StepCreateAndUpdateKeys, StepIssue>;
```

Comment:
love it!  
did you make sure that no one is using the `body` param?

Reply:
We need too :P

Reply:
The only consumer is hte Dashboard, so I think we should be fine.